pkg/webhook/namespacelabel.go

```diff
@@ -0,0 +1,85 @@
+package webhook
+
+import (
+	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"net/http"
+
+	opa "github.com/open-policy-agent/frameworks/constraint/pkg/client"
+	"github.com/pkg/errors"
+	types "k8s.io/api/admission/v1beta1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+)
+
+var (
+	exemptNamespaces = newNSSet()
+)
+
+func init() {
+	AddToManagerFuncs = append(AddToManagerFuncs, AddLabelWebhook)
+	flag.Var(exemptNamespaces, "exempt-namespaces", "Namespaces that are allowed to set the admission.gatekeeper.sh/ignore label. To exempt multiple namespaces, this flag can be declared more than once.")
```

Comment:
should we default to `gatekeeper-system` here?

Reply:
I thought about it, but decided against it because:

* Some people had wanted to deploy gatekeeper into a different namespace, defaulting the flag risks a misconfiguration that leaves a security hole (could maybe be handled by grabbing the namespace from the environment)
* By keeping all configuration in the YAML we avoid having the problems we've been having with changes to the YAML and changes to the code being out of sync: if the namespaces or defaults change, there is only one place to look
* Principle of least privilege: assume people don't want to exempt any namespaces unless they specifically ask
* No need to explain/figure out "what happens to the default when I add more namespaces? Is it preserved or do I explicitly re-add it?" and all the breaking edge cases that would follow encountering these

Reply:
ok sounds good